WalletWasabi.Fluent/ViewModels/Wallets/Settings/WalletCoinJoinSettingsViewModel.cs

```diff
@@ -27,13 +30,18 @@ public partial class WalletCoinJoinSettingsViewModel : RoutableViewModel
 	[AutoNotify] private bool _isCoinjoinProfileSelected;
 	[AutoNotify] private string _plebStopThreshold;
 	[AutoNotify] private string? _selectedCoinjoinProfileName;
+	[AutoNotify] private Wallet? _selectedOutputWallet;
+	[AutoNotify] private ObservableCollection<Wallet> _wallets;
 
 	private WalletCoinJoinSettingsViewModel(IWalletModel walletModel)
 	{
 		_wallet = walletModel;
 		_autoCoinJoin = _wallet.Settings.AutoCoinjoin;
 		_plebStopThreshold = _wallet.Settings.PlebStopThreshold.ToString();
 		_anonScoreTarget = _wallet.Settings.AnonScoreTarget;
+		_selectedOutputWallet = _wallet.Settings.OutputWallet;
+
+		_wallets = new ObservableCollection<Wallet>(Services.WalletManager.GetWallets());
```

Comment:
Static calls are forbidden on VIewModdel level. `UiContext.WalletRepository` has the list of wallets.

Reply:
Done